l10n_es_aeat_mod390_vat_prorrate/models/mod390_line_prorrate.py

```diff
@@ -0,0 +1,53 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+
+from odoo import api, fields, models
+
+
+class L10nEsAeatMod390ReportLineProrrate(models.Model):
+    _name = 'l10n.es.aeat.mod390.report.line.prorrate'
+    _description = 'AEAT 390 report line prorrate'
+
+    report_id = fields.Many2one(
+        string="Report 390",
+        comodel_name="l10n.es.aeat.mod390.report",
+        readonly=True
+    )
+    activity = fields.Char(
+        string="Activity",
+        required=True
+    )
+    cnae = fields.Char(
+        string="CNAE",
+        size=3,
+        required=True
+    )
+    amount = fields.Float(
+        string="Total amount of operations",
+    )
+    amount_allow_deduction = fields.Float(
+        string="Amount allow deduction",
+    )
+    type = fields.Selection(
+        [
+            ('general', 'General')
+        ],
+        string="Type",
+        default='general',
+        required=True
+    )
+    percent = fields.Float(
+        string="Percent",
+        default=100
+    )
+    amount_prorrate = fields.Float(
+        string="Prorrate",
+        readonly=True,
+        compute="_compute_amount_prorrate",
+        store=True
+    )
+
+    @api.multi
```

Comment:
```suggestion
```

Reply:
In OCA Guidelines about 12.0 https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0 not appear this change, maybe it's necessary